documentation/Makefile

```diff
@@ -2,7 +2,7 @@
 #
 
 # You can set these variables from the command line.
-SPHINXOPTS    =
+SPHINXOPTS    = -vn
```

Comment:
Is it intended these options remain in master to help with debugging and fixing things?

Reply:
Presumably, leaving for him to answer. Ideally it would be `?=` to allow command line overrides, but not particularly important.

Reply:
Yes, the documentation basically acts as a test suite as well, so I found it useful to have full stack traces and nitpicky mode on all the time (though I do not necessarily recommend end users to do this as default). I have made the options overridable as suggested.